src/app/modules/group/components/pending-request/pending-request.component.html

```diff
@@ -16,8 +16,8 @@
   <div class="pending-request-container" *ngIf="status !== 'loading'; else loading" [ngSwitch]="status">
     <alg-grid
       *ngSwitchCase="'loaded'"
-      [selectedColumns]="columns"
-      [columns]="columns"
+      [selectedColumns]="getGroupGrid()"
+      [columns]="getGroupGrid()"
```

Comment:
You cannot depend on a function in Angular templates, otherwise they are called hundred times, each time angular has to refresh anything on the view.

Reply:
fixed